Partest should fail nice on empty dir

On an empty dir, instead of a nice error or ignoring it,

```
java.lang.RuntimeException: Error running /home/apm/clones/scala/test/files/pos/AAAA
	at scala.tools.partest.nest.SuiteRunner.liftedTree1$1(Runner.scala:773)
	at scala.tools.partest.nest.SuiteRunner.runTest(Runner.scala:771)
	at scala.tools.partest.nest.SuiteRunner$$anonfun$19$$anonfun$apply$6.apply(Runner.scala:786)
	at scala.tools.partest.nest.SuiteRunner$$anonfun$19$$anonfun$apply$6.apply(Runner.scala:786)
	at scala.tools.partest.package$$anon$2.call(package.scala:133)
	at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:334)
	at java.util.concurrent.FutureTask.run(FutureTask.java:166)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:724)
Caused by: java.lang.UnsupportedOperationException: tail of empty list
```

--- a/src/partest/scala/tools/partest/nest/Runner.scala
+++ b/src/partest/scala/tools/partest/nest/Runner.scala
@@ -437,7 +437,7 @@ class Runner(val testInfo: TestInfo, val suiteRunner: AbstractRunner) {
 
   /** Grouped files in group order, and lex order within each group. */
   def groupedFiles(sources: List[File]): List[List[File]] = (
-    if (sources.tail.nonEmpty) {
+    if (sources.sizeIs > 1) {
       val grouped = sources groupBy (_.group)
       grouped.keys.toList.sorted map (k => grouped(k) sortBy (_.getName))
     }

